packages/multi-trait-rubric/configure/src/__tests__/index.test.jsx

```diff
@@ -0,0 +1,530 @@
+import MultiTraitRubricElement from '../index';
+import defaults from '../defaults';
+import { ModelUpdatedEvent, InsertSoundEvent, DeleteSoundEvent } from '@pie-framework/pie-configure-events';
+import React from "react";
+
+jest.mock('react-dom', () => ({
+    render: jest.fn(),
+}));
+
+const model = (extras) => ({
+    id: '1',
```

Comment:
Let's reduce the amount of information here. You miht not need all the model properties.
Another suggestion is to add more clear labels, like "Trait 1", "Trait 2" etc.